### PR TITLE
19378: CT 116: Final MVP Content Updates based on UX Feedback

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to make the final pre-launch pass of content updates so that the user has a consistent experience and is provided with accurate and complete information.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19378

## Testing done
Local manual, e2e, and unit tests pass.

## Screenshots
image
image
image

## Acceptance criteria
- [x] 1. On the VET TEC Search Results, the "Learning Format" filter label text is formatted as: "Learning format"
The filter option "In Person" label text is formatted as: "In person"
- [x] 2. On the VET TEC Search Results, the card text "Tuition (annually):" label text is formatted as: "Tuition:"
- [x] 3. On the VET TEC Profile Page the "Out of Pocket Cost" calculates the cost based on the following rules:
If the user enters nothing in the tuition field, "TBD" is displayed.
If the user enters tuition greater than the cap and the provider is not preferred, the amount displayed is: Tuition amount entered - T&F Cap (TFCAP Constant) - Scholarships entered.
If the user enters tuition less than the cap and the provider is not preferred, the amount displayed is: $0
- [x] 4. If the user enters tuition and the provider is preferred, the amount displayed is: $0
- [x] 5. If the user clears out / sets the tuition field to $0, all calculated fields in the calculator are reset to "TBD".
If the user only enters a scholarship amount, all calculated fields in the calculator are set to "TBD".

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Simplify determineOutOfPocketFees.

File: src/applications/gi/selectors/vetTecCalculator.js [hunk 1]
```diff
@@ -13,29 +14,53 @@ export const getCalculatedBenefits = createSelector(
   getConstants,
   getInstitution,
   (inputs, constants, institution) => {
-    const inputsNull =
-      inputs.vetTecTuitionFees === null && inputs.vetTecScholarships === null;
+    const vetTecTuitionInputNull =
```
Comment (zurbergram) [inline src/applications/gi/selectors/vetTecCalculator.js]: Does this need to be renamed?

Comment (zurbergram) [inline src/applications/gi/selectors/vetTecCalculator.js]: Let's do a pluck on all the fields from `inputs`.

File: src/applications/gi/selectors/vetTecCalculator.js [hunk 2]
```diff
@@ -13,29 +14,53 @@ export const getCalculatedBenefits = createSelector(
...
+      inputs.vetTecTuitionFees === null || inputs.vetTecTuitionFees === 0;
```
Comment (zurbergram) [inline src/applications/gi/selectors/vetTecCalculator.js]: You are removing the check `&& inputs.vetTecScholarships === null` from this, why?

File: src/applications/gi/selectors/vetTecCalculator.js [hunk 3]
```diff
@@ -13,29 +14,53 @@ export const getCalculatedBenefits = createSelector(
...
       : paysToProviderValue * 0.25;
 
-    const halfPaysToProviderValue = inputsNull
+    const halfPaysToProviderValue = vetTecTuitionInputNull
       ? null
       : paysToProviderValue * 0.5;
 
+    const determineOutOfPocketFees = () => {
+      let outOfPocketFees;
+      if (vetTecTuitionInputNull) {
```
Comment (zurbergram) [inline src/applications/gi/selectors/vetTecCalculator.js]: Change this to
```suggestion
      if (vetTecTuitionFees === null ||vetTecTuitionFees === 0) {
```

File: src/applications/gi/selectors/vetTecCalculator.js [hunk 4]
```diff
@@ -13,29 +14,53 @@ export const getCalculatedBenefits = createSelector(
...
+    const quarterPaysToProviderValue = vetTecTuitionInputNull
       ? null
       : paysToProviderValue * 0.25;
 
-    const halfPaysToProviderValue = inputsNull
+    const halfPaysToProviderValue = vetTecTuitionInputNull
       ? null
       : paysToProviderValue * 0.5;
 
+    const determineOutOfPocketFees = () => {
```
Comment (zurbergram) [inline src/applications/gi/selectors/vetTecCalculator.js]: Remove this as a function just do this as a part of `getCalculatedBenefits`

**Review by zurbergram — COMMENTED** [review]
Please make requested changes

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**